src/components/GenreSelect/GenreSelect.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import './GenreSelect.css'; // Importa el archivo CSS
 
-export function GenreSelect({ genders , current, selectedGenreCallback }) {
+export function GenreSelect({ genres , current, selectedGenreCallback }) {
   const [selectedGenre, setSelectedGenre] = useState(current);
```

Comment:
nit: You could keep track of this value in the App or outside component. The `current` attribute sounds like should keep track of the selected genre. In this case, it's only being used as a `initialValue` not `current`. So, according to the spec:
> A name of currently selected genre. Use the name to identify which button to highlight.
This attribute should be the current value, not the initial value. Feel free to update the logic and keep the state outside of the component

```
const App = () => {
  const [currentGenre, setCurrentGenre] = useState();

  return (
    <GenreSelect genres={genres} current={currentGenre} onSelect={(selected) => setCurrentGenre(selected)} />
  )
}
```